docs/concepts/services-networking/manging-hosts-file-with-host-aliases.md

```diff
@@ -0,0 +1,71 @@
+---
+assignees:
+- rickypai
+- thockin
+title: Manging Hosts File with HostAliases
```

Comment:
Can we change this to "Adding entries to pod `/etc/hosts` with HostAliases" which is more direct and searchable (e.g. someone looking for the "/etc/hosts" will be able to find the title).

Manging => Managing